src/hotspot/share/opto/superword.cpp

```diff
@@ -3941,7 +3941,7 @@ void SuperWord::compute_vector_element_type() {
             // vectorized if the signedness info is imprecise.
             const Type* vt = vtn;
             int op = in->Opcode();
-            if (VectorNode::is_shift_opcode(op) || op == Op_AbsI) {
+            if (VectorNode::is_shift_opcode(op) || op == Op_AbsI || op == Op_ReverseBytesI) {
```

Comment:
(another suggestion) This list may be longer and longer as we vectorize more operations. Shall we move this check into a static function of `VectorNode`, like `VectorNode::requires_higher_order_bits(op)`, and put the comments inside?

Reply:
Good suggestion, I think a function can make this condition self-documented. Thank you.

Reply:
I have made the changes but I leave the updated comments here. This comment seems to be more related to the context here instead of that new function. Could you please review it again? Thanks.